pkg/bootstrapper/bootstrapper.go

```diff
@@ -554,6 +554,7 @@ func (wmcb *winNodeBootstrapper) generateInitialKubeletArgs(args map[string]stri
 		"--image-pull-progress-deadline=30m",
 		"--container-runtime=remote",
 		"--container-runtime-endpoint=" + containerdEndpointValue,
+		"--resolv-conf=" + resolvValue,
```

Comment:
I think this should always be set to ""
It is supposed to point to a file, but by passing it the expected path on rhcos VMs its causing errors.

Reply:
yes it is being set to "" here, that's the resolvValue used. If we do not do this then the default C:\k\resolve.conf path is taken which leads to errors during sandbox creation.